internal/pkg/generator/forwarder/generator.go

```diff
@@ -18,15 +20,22 @@ const (
 	useOldRemoteSyslogPlugin = false
 )
 
-func Generate(clfYaml string, includeDefaultLogStore, includeLegacyForward bool) (string, error) {
+func Generate(clfYaml string, includeDefaultLogStore, includeLegacyForward, debug_output bool) (string, error) {
```

Comment:
please rename debug_output to debugOutput

Reply:
changed